### DISCUSSION TITLE
Hybrid & Keyword Queries

### DISCUSSION THREAD

**Discussion by mikehudson2** [FlowiseAI/Flowise#1962] [Q&A]
Some VDBs support 'hybrid' searching - using keywords/sparse vectors - for example, Pinecone and Weaviate. Do any of the Flowise nodes either support keyword queries or hybrid queries?

**Comment by Servin2** [comment]
make pull reuest for this its good suggestion

**Comment by dan71888** [comment]
My developer and I have been struggling to get relevant responses from our FlowiseAI bot becuase it seems to only be working for semantic searches and not keyword. Both are extremely valuable. Please Flowise release a hybrid option for both semantic and keyword search!!

**Reply by gururise** [reply]
Hey @dan71888 

I just added hybrid search to Weaviate in my repo: https://github.com/gururise/Flowise/tree/weaviate

Try it out and let me know if its working for you. If so, I'll make a PR.
